src/main/java/viper/gobra/frontend/LICENSE

```diff
@@ -0,0 +1,5 @@
+This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is here because the generated parser won't have the license header. The Mozilla license allows placing license file in the same directory for cases like this.